blocks/editable/format-toolbar/index.js

```diff
@@ -56,6 +58,10 @@ class FormatToolbar extends Component {
 		this.onChangeLinkValue = this.onChangeLinkValue.bind( this );
 		this.toggleLinkSettingsVisibility = this.toggleLinkSettingsVisibility.bind( this );
 		this.setLinkTarget = this.setLinkTarget.bind( this );
+
+		this.addFootnote = this.addFootnote.bind( this );
```

Comment:
Related: #2474

This is _begging_ to be extensible behavior, not engrained into the toolbar itself.